test/integration/modules/edgecore/eventbus/eventbus_test.go

```diff
@@ -0,0 +1,203 @@
+			Client = MQTT.NewClient(ClientOpts)
+			if Token_client = Client.Connect(); Token_client.Wait() && Token_client.Error() != nil {
+				common.Failf("client.Connect() Error is %s", Token_client.Error())
+			}
+			Expect(Token_client.Error()).NotTo(HaveOccurred())
+		})
+		AfterEach(func() {
+			common.PrintTestcaseNameandStatus()
+		})
+
+		It("TC_TEST_EBUS_1: Sending data to Cloud", func() {
+			if Token_client = Client.Publish(UploadRecordToCloud, 0, false, "messagetoUpload_record_to_cloud"); Token_client.Wait() && Token_client.Error() != nil {
+				common.Failf("client.Publish() Error is %s", Token_client.Error())
+			} else {
+				common.InfoV6("client.Publish Success !!")
+			}
+			Expect(Token_client.Error()).NotTo(HaveOccurred())
+			Client.Disconnect(1)
+		})
+
+		It("TC_TEST_EBUS_2: Sending data to device module", func() {
+			if Token_client = Client.Publish(DevicestatusUpdate, 0, false, "messagetoDevice_status_update"); Token_client.Wait() && Token_client.Error() != nil {
+				common.Failf("client.Publish() Error is %s", Token_client.Error())
+			} else {
+				common.InfoV6("client.Publish Success !!")
+			}
+			Expect(Token_client.Error()).NotTo(HaveOccurred())
+			Client.Disconnect(1)
+		})
+
+		It("TC_TEST_EBUS_3: Sending data to device twin module", func() {
+			if Token_client = Client.Publish(DeviceTwinUpdate, 0, false, "messagetoDevice_Twin_update"); Token_client.Wait() && Token_client.Error() != nil {
+				common.Failf("client.Publish() Error is %s", Token_client.Error())
+			} else {
+				common.InfoV6("client.Publish Success !!")
+			}
+			Expect(Token_client.Error()).NotTo(HaveOccurred())
+			Client.Disconnect(1)
+		})
+
+		It("TC_TEST_EBUS_4: Sending data to membership module", func() {
+			if Token_client = Client.Publish(DeviceMembershipUpdate, 0, false, "messagetoDevice_Membership_update"); Token_client.Wait() && Token_client.Error() != nil {
+				common.Failf("client.Publish() Error is %s", Token_client.Error())
+			} else {
+				common.InfoV6("client.Publish Success !!")
+			}
+			Expect(Token_client.Error()).NotTo(HaveOccurred())
+			Client.Disconnect(1)
+		})
+
+		It("TC_TEST_EBUS_5: Sending data to device event module", func() {
+			if Token_client = Client.Publish(DeviceUpload, 0, false, "messagetoDevice_upload"); Token_client.Wait() && Token_client.Error() != nil {
+				common.Failf("client.Publish() Error is %s", Token_client.Error())
+			} else {
+				common.InfoV6("client.Publish Success !!")
+			}
+			Expect(Token_client.Error()).NotTo(HaveOccurred())
+			Client.Disconnect(1)
+		})
+
+		It("TC_TEST_EBUS_6: change the device status to online from eventbus", func() {
+			var message DeviceUpdate
+			message.State = "online"
+			body, _ := json.Marshal(message)
+			deviceId := ctx.Cfg.DeviceId
+
+			if Token_client = Client.Publish("$hw/events/device/"+deviceId+"/state/update", 0, false, body); Token_client.Wait() && Token_client.Error() != nil {
```

Comment:
use existing variables like above code. applies to all such cases.

Reply:
@rohitsardesai83 , Yes its updated.